<launch>

  <arg name="x" default="0.5" />
  <arg name="y" default="0.5" />
  <arg name="angle" default="0" />

  <node name="excel_pick_bin" pkg="excel_bins" type="pick_bin_node" output="screen">
	<param name="x_goal" value="$(arg x)" />
  	<param name="y_goal" value="$(arg y)" />
  	<param name="angle" value="$(arg angle)" />
  </node>

</launch>
